src/components/general/CausalGrid.tsx

```diff
@@ -386,40 +420,64 @@ const CausalGrid = ({
               </>
             )}
           </StyledShowCalculationButton>
-        </ArcherElement>
-        <div id="causal-grid" aria-hidden={!gridExpanded}>
-          {causalGridNodes?.map((row, rowIndex) => (
-            <GridRowWrapper onScroll={() => gridCanvas.current?.refreshScreen()} key={rowIndex}>
-              <GridRow>
-                {row.map((col) => (
-                  <GridCol key={col.id}>
-                    <ArcherElement
-                      id={col.id}
-                      relations={col.outputNodes
-                        .filter((outnode) => [lastNode.id, ...visibleNodesIds].includes(outnode.id))
-                        .map((node) => ({
-                          targetId: node.id,
-                          targetAnchor: 'top',
-                          sourceAnchor: 'bottom',
-                        }))}
-                    >
-                      <div>
-                        <CausalCard
-                          node={col}
-                          compact={false}
-                          startYear={yearRange[0]}
-                          endYear={yearRange[1]}
-                          noEffect={actionIsOff}
-                        />
-                      </div>
-                    </ArcherElement>
-                  </GridCol>
-                ))}
-              </GridRow>
-            </GridRowWrapper>
-          ))}
+          {/* Visually hidden region to announce state by screen-readers*/}
+          <span style={{position:'absolute',clip:'rect(1px,1px,1px,1px)'}} aria-live="polite">
```

Comment:
There's also a MUI utility for this if you'd like to avoid inline styles

https://mui.com/system/screen-readers/#visually-hidden-elements